json/uk.json

```diff
@@ -1,186 +1,186 @@
+  "This password reset link will expire in :count minutes.": "Термін дії цього посилання для скидання пароля закінчується через :count хвилин.",
+  "This user already belongs to the team.": "Цей користувач вже належить до команди.",
+  "to" : "до",
+  "Toggle navigation": "Перемкнути меню",
+  "Token Name" : "Назва токена",
+  "Too Many Attempts.": "Занадто багато спроб.",
+  "Too Many Requests": "Забагато запитів",
+  "Two factor authentication is now enabled. Scan the following QR code using your phone's authenticator application." : "Двофакторна автентифікація тепер увімкнена. Відскануйте наступний QR-код за допомогою програми автентифікації вашого телефону.",
+  "Two Factor Authentication" : "Двофакторна аутентифікація",
+  "Unauthorized": "Неавторизований",
+  "Update Password" : "Оновити пароль",
+  "Update your account's profile information and email address." : "Оновіть дані профілю свого облікового запису та електронну адресу.",
+  "Use a recovery code": "Використовуйте код відновлення",
+  "Use an authentication code": "Використовуйте код автентифікації",
+  "Verify Email Address": "Підтвердьте адресу електронної пошти",
+  "Verify Your Email Address": "Підтвердьте свою адресу електронної пошти",
+  "We were unable to find a registered user with this email address.": "Нам не вдалося знайти зареєстрованого користувача з цією електронною адресою.",
+  "We won't ask for your password again for a few hours.": "Ми не будемо запитувати ваш пароль знову протягом кількох годин.",
+  "When two factor authentication is enabled, you will be prompted for a secure, random token during authentication. You may retrieve this token from your phone's Google Authenticator application." : "Коли увімкнено двофакторну автентифікацію то під час автентифікації вам буде запропоновано ввести безпечний випадковий токен. Ви можете отримати цей токен із програми Google Authenticator на своєму телефоні.",
+  "Whoops, something went wrong on our servers.": "Ой, щось пішло не так на наших серверах.",
+  "Whoops! Something went wrong." : "Ой! Щось пішло не так.",
+  "Whoops!": "Ой!",
+  "You are logged in!": "Ви увійшли в систему",
+  "You are receiving this email because we received a password reset request for your account.": "Ви отримали цей електронний лист, оскільки ми отримали запит на скидання паролю для вашого облікового запису.",
+  "You have enabled two factor authentication." : "Ви ввімкнули двофакторну автентифікацію.",
+  "You have not enabled two factor authentication." : "Ви не ввімкнули двофакторну автентифікацію.",
+  "You may delete any of your existing tokens if they are no longer needed." : "Ви можете видалити будь-який з існуючих токенів, якщо вони більше не потрібні.",
+  "You may not delete your personal team.": "Ви не можете видалити свою особисту команду.",
+  "You may not leave a team that you created.": "Ви не можете залишити створену вами команду.",
+  "Your email address is not verified.": "Ваша електронна адреса не підтверджена."
```

Comment:
In this project, we are working with an [**indent size**](https://github.com/Laravel-Lang/lang/blob/master/.editorconfig#L12) of *4 spaces*, you changed this to 2 spaces.